blog/_posts/2017-09-24-we-heart-summer-of-code.md

```diff
@@ -0,0 +1,79 @@
+---
+title: We ❤️ Rails Girls Summer of Code
+layout: post
+created_at: Sunday 24 September 2017
+permalink: blog/2017-09-24-we-heart-summer-of-code
+current: blog
+author: Emma Deacon
+twitter: emmamdeacon
+categories:
+- student-posts
+---
+
+*This post contains songs we recommend listening to whilst reading - we felt that this incredible summer needed an incredible soundtrack to tell its story*
+
+
+We have had an incredible summer: it has been insanely busy and at times overwhelming, but we have both grown enormously as developers and team members. Rails Girls Summer of Code was one of the best decisions we have ever made, and it’s been a pleasure to be part of this superb and inspirational community. Being part of Rails Girls Summer of Code is an incredible opportunity, and it has been a transformative summer for us.
```

Comment:
Hi, thanks for the PR. Early PRs are always welcome. Chekout the comments below :)